docs/docs/examples/shared-module.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Shared module
+description: How to share modules across code locations.
+last_update:
+  author: Dennis Hume
+sidebar_custom_props:
+  logo: images/dagster-primary-mark.svg
+  miniProject: true
+tags: [mini-project]
+---
+
+In this example, we'll explore strategies for sharing code across Dagster code locations. This is useful when you have utility functions, factories, or helpers that are used in multiple places and you want to avoid duplication.
```

Comment:
```suggestion
In this example, we'll explore strategies for sharing code across Dagster [code locations](/deployment/code-locations). This is useful when you have utility functions, factories, or helpers that are used in multiple places and you want to avoid duplication.
```